_build_cfg.yml

```diff
@@ -210,6 +208,8 @@ Topics:
     File: syncing_groups_with_ldap
   - Name: Available Ports
     File: available_ports
+  - Name: Aggregating Logs with the EFK Stack
+    File: enabling_EFK
```

Comment:
Whatever the topic title ends up being, the file name should probably be renamed to match it better.